jackson-jr-test-module/pom.xml

```diff
@@ -0,0 +1,112 @@
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.jr</groupId>
+            <artifactId>jackson-jr-objects</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>${jackson.version.core}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.groovy</groupId>
+            <artifactId>groovy</artifactId>
+            <version>4.0.18</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <configuration>
+                    <links combine.children="append">
+                        <link>https://javadoc.io/doc/com.fasterxml.jackson.core/jackson-core/2.17/</link>
+                    </links>
+                </configuration>
+
+            </plugin>
+
+            <!-- 20-Feb-2024, tatu: Need Groovy compilation for one test
+          -->
+            <plugin>
+                <groupId>org.codehaus.gmaven</groupId>
+                <artifactId>gmaven-plugin</artifactId>
+                <version>1.5</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.groovy</groupId>
+                        <artifactId>groovy</artifactId>
+                        <version>4.0.18</version>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <providerSelection>2.0</providerSelection>
+                    <source>2.0</source>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>testCompile</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.google.code.maven-replacer-plugin</groupId>
+                <artifactId>replacer</artifactId>
+            </plugin>
+            <!-- 11-Mar-2019, tatu: Add basic JDK8-includable module-info, generated by Moditect -->
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+            </plugin>
+            <!-- 05-Jul-2020, tatu: Add generation of Gradle Module Metadata -->
+            <plugin>
+                <groupId>de.jjohannes</groupId>
+                <artifactId>gradle-module-metadata-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```

Comment:
Could you ensure line endings at end of every file?

Reply:
I tried to fix the formatting of this file, but i don't see any problem/change in it, if you are using a formatter, can you share with me, so that this can be avoided in future.

Reply:
I think the end-of-file linefeed is sort of typical expectation, outside of other formatting aspects (some tools complaint about it).
I'll go ahead and fix this.

Reply:
Looks like there is trailing linefeed, all good.